ts/screens/profile/components/SettingsDiscoveryBanner.tsx

```diff
@@ -0,0 +1,45 @@
+import { Banner, IOVisualCostants } from "@pagopa/io-app-design-system";
+import React, { createRef, useState } from "react";
+import { StyleSheet, View } from "react-native";
+import I18n from "../../../i18n";
+import { useIONavigation } from "../../../navigation/params/AppParamsList";
+import ROUTES from "../../../navigation/routes";
+
+export const SettingsDiscoveryBanner = () => {
+  const bannerRef = createRef<View>();
+  const navigation = useIONavigation();
+  const [isVisible, setVisible] = useState(true);
```

Comment:
Should this component be always visibile when we restart the app or, once closed, we do not want to present it anymore (always/or since last new login)?

Reply:
Thank you for your comment, I have added the logic to show the user this banner only if they have not closed it (as required by the specification) using a persistent status on redux. Please check the PR again and provide me with more comments if there are some.